fury/tests/test_actors.py

```diff
@@ -1176,6 +1176,7 @@ def timer_callback(_obj, _event):
     arr[arr < 20] = 0
     report = window.analyze_snapshot(arr)
     npt.assert_equal(report.objects, 6)
+    npt.assert_equal(type(container._items[0]), actor.Container)
```

Comment:
Please avoid using private members, instead use the property.

Reply:
Done!